routes/web.php

```diff
@@ -49,6 +49,11 @@
     });
 });
 
+Route::group(['prefix' => 'beatmappacks'], function () {
+    Route::get('listing/{type?}', 'BeatmappacksController@index')->name('beatmappacks.index');
```

Comment:
more like `BeatmapPacksController` because it explodes on case sensitive filesystem.